lib/howl/commands/file_commands.moon

```diff
@@ -70,14 +78,19 @@ command.alias 'save', 'w'
 command.register
   name: 'save-as',
   description: 'Saves the current buffer to a given file'
-  input: 'file'
-  handler: (file) ->
+  interactive: true
+  evade_history: true
+  handler: ->
+    file = interact.select_file allow_new: true
+    return unless file
     if file.exists
-      input = inputs.yes_or_no false
-      unless inputs.read input, prompt: "File '#{file}' already exists, overwrite? "
+      unless interact.yes_or_no prompt: "File '#{file}' already exists, overwrite? "
         log.info "Not overwriting; buffer not saved"
         return
 
+    unless file.parent.exists
```

Comment:
I'm mostly concerned about the consistency vs `save`, it seems like if one of these asks for confirmation than both should do so since the operation performed is the same.